lms/djangoapps/courseware/views.py

```diff
@@ -6,13 +6,14 @@
 import urllib
 import json
 import cgi
-
 from datetime import datetime
 from collections import defaultdict
+from markupsafe import escape
```

Comment:
Actually, all of the changes in this file appear to be unnecessary stylistic cleanups.